modules/auth/params.go

```diff
@@ -52,7 +52,7 @@ func (p *Params) Validate(key string, value string) (interface{}, sdk.Error) {
 		if !ok {
 			return nil, params.ErrInvalidString(value)
 		}
-		if !threshold.GT(MinimumGasPrice) || threshold.GT(MaximumGasPrice) {
+		if threshold.LT(MinimumGasPrice) || threshold.GT(MaximumGasPrice) {
```

Comment:
GasPriceThreshold range is (0, 1iris]. So the threshold can not be equal with MinimumGasPrice.